wp-content/themes/borderzine/partials/widget-content.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Custom partials/widget-content.php
+ *
+ * Modified thusly:
+ * - replace the post thumbnail with a play button on the podcast widget
+ *
+ * @since Largo 0.6.4 - when this file was refreshed from Largo
+ */
+
+// the thumbnail image (if we're using one)
+if ( $podcast === true ) {
```

Comment:
This is affected by https://github.com/INN/umbrella-citylimits/issues/87:

1. Put this podcast widget in a widget area.
2. Put a Largo Recent Posts widget after it.
3. The Largo Recent Posts widget displays the play icon.

Reply:
Right. So we'll probably need to add `podcast => false` to all of the other widget `$context` vars like in CL

Reply:
If we're not mixing Largo widgets with Borderzine widgets, that's fine, but if we're mixing Largo widgets in, we're going to have to reset the variable within this widget.

Maybe we can use  `largo_render_template( 'partials/widget', 'content', $context );` a second time with a partial that has no content and a `$context` with `'podcast' => false`?

Reply:
@benlk, https://github.com/INN/umbrella-borderzine/pull/77/commits/1135db9b5858f8b5155ca7ed67ea1c5bbab92988 / https://github.com/INN/umbrella-borderzine/pull/77/commits/9314ed83b52a6e656d45689885ffc357876a8fd9 should do the trick. It renders another empty partial at the very end of the widget and it sets `'podcast' => false`. 

Also modified `/partials/widget-content.php` to make sure `$thumb` is set before trying to use it, because an `undefined` error was being thrown from it trying to render from the context with no content.

Example of podcast widget with Largo Recent Posts under it:
![Screen Shot 2019-12-06 at 12 56 32 PM](https://user-images.githubusercontent.com/18353636/70344479-d71cc700-1827-11ea-815a-29d23c6e8ba2.png)

Reply:
We'll need to make sure `$podcast` is set as well in this partial:

![Screen Shot 2019-12-06 at 15 17 38](https://user-images.githubusercontent.com/1754187/70353561-a050ac00-183b-11ea-8807-0b7708265d78.png)

```diff
- if ( $podcast === true ) {
+ if ( isset( $podcast) && $podcast === true ) {
```